src/gengowatcher/logging_setup.py

```
@@ -93,7 +94,16 @@ def emit(self, record):
             f"{datetime.datetime.fromtimestamp(record.created).strftime('%H:%M:%S')} - "
             f"{record.getMessage()}"
         )
-        self.log_queue.append(Text.from_markup(message, style=style))
+
+        # Try to parse markup safely, fall back to sanitized plain text
+        try:
+            text_obj = Text.from_markup(message)
+        except Exception:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)